stdlib/REPL/test/replcompletions.jl

```diff
@@ -158,6 +158,10 @@ let ex =
             export exported_symbol
             exported_symbol(::WeirdNames) = nothing
 
+            macro ignoremacro(e...)
+                :nothing
```

Comment:
Should this be
```suggestion
                nothing
```
?
```julia-repl
julia> macro ignoremacro(e...)
           :nothing
       end
@ignoremacro (macro with 1 method)

julia> @macroexpand @ignoremacro
:(Main.nothing)

julia> macro ignoremacro(e...)
           nothing
       end
@ignoremacro (macro with 1 method)

julia> @macroexpand @ignoremacro

julia> 
```

Reply:
I was actually unaware that so many things were self-quoting!